### DISCUSSION TITLE
How can I update my server from Linux command line?

### DISCUSSION THREAD

**Discussion by AntonioPietrok** [wolveix/satisfactory-server#381] [Q&A] [answered]
I downloaded this docker container a couple days ago and got a server up and running. Today I had a game update and now the in game server manager says "Version mismatch detected between game and server. Game Version: 383729, Sever Version: 382498". It does not allow me to to access any of the setting or the command line in game. I have access to the docker container/sever files from the linux command, is there a way I can use this to update my server?

**Comment by erockFTW** [comment, marked as answer]
You just need to restart the container. It will update the server as the container starts.

**Reply by AntonioPietrok** [reply]
Thank you so much. I know it's silly but I spent way to long trying to figure this out last night.

**Reply by wolveix** [reply]
@AntonioPietrok you're far from the first person to ask! I'll add clearer wording in the README to better document this :)
